src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/ForgotPassword.razor.cs

```diff
@@ -1,72 +1,34 @@
-﻿using System.Text.RegularExpressions;
+﻿using TodoTemplate.App.Models;
 
 namespace TodoTemplate.App.Pages;
 
 public partial class ForgotPassword
 {
-    public string? Email { get; set; }
+    public ForgotPasswordModel ForgotPasswordModel { get; set; } = new();
 
-    public string? EmailErrorMessage { get; set; }
-
-    public bool IsSubmitButtonEnabled { get; set; }
     public bool IsLoading { get; set; }
 
-    [Inject] public HttpClient HttpClient { get; set; } = default!;
-
     public BitMessageBarType ForgotPasswordMessageType { get; set; }
-    public string? ForgotPasswordMessage { get; set; }
-
-    private void CheckSubmitButtonEnabled()
-    {
-        if (string.IsNullOrEmpty(Email))
-        {
-            IsSubmitButtonEnabled = false;
-            return;
-        }
-        IsSubmitButtonEnabled = true;
-    }
-
-    private bool ValidateForgotPassword()
-    {
-        EmailErrorMessage = string.IsNullOrEmpty(Email)
-            ? "Please enter your email."
-            : null;
-
-        if (string.IsNullOrEmpty(EmailErrorMessage) is false)
-        {
-            return false;
-        }
-
-        var isCorrectEmailFormat = new Regex(@"^([\w\.\-]+)@([\w\-]+)((\.(\w){2,3})+)$");
-
-        EmailErrorMessage = isCorrectEmailFormat.Match(Email!).Success is false
-            ? "The email address format is incorrect."
-            : null;
 
-        if (string.IsNullOrEmpty(EmailErrorMessage) is false)
-        {
-            return false;
-        }
+    public string? ForgotPasswordMessage { get; set; }
 
-        return true;
-    }
+    [Inject] public HttpClient HttpClient { get; set; } = default!;
 
-    private async Task Submit()
+    private async Task DoForgotPassword()
```

Comment:
plz rename this method back to submit. the `DoForgotPassword` is not an appropriate name here.